components/PlantList.js

```diff
@@ -1,17 +1,30 @@
 import PlantsPreview from "./PlantsPreview";
+import styled from "styled-components";
+
+const StyledUL = styled.ul`
```

Comment:
choose a more descriptive name for the element's role as a container for the list of plants. It immediately tells you what the component is for without having to see the styled properties. 



```suggestion
const PlantsListContainer = styled.ul`
```